sources/install/package_network.sh

```diff
@@ -92,6 +92,17 @@ function install_nmap() {
     add-to-list "nmap,https://nmap.org,The Network Mapper - a powerful network discovery and security auditing tool"
 }
 
+function install_nmap-parse-output() {
+    colorecho "Installing nmap-parse-output"
+    fapt xsltproc
+    git -C /opt/tools/ clone --depth 1 https://github.com/ernw/nmap-parse-output
+    add-aliases nmap-parse-output
```

Comment:
Please remove this, we'll do a symbolic link instead `ln -s /opt/tools/nmap-parse-output/nmap-parse-output /opt/tools/bin/nmap-parse-output`